leakcanary-android-core/src/main/java/leakcanary/DefaultOnRetainInstanceListener.kt

```diff
@@ -0,0 +1,22 @@
+package leakcanary
+
+import leakcanary.internal.InternalLeakCanary
+import leakcanary.internal.InternalLeakCanary.FormFactor.TV
+import leakcanary.internal.tv.TvOnRetainInstanceListener
+
+/**
+ * Default implementation of [OnRetainInstanceListener] that doesn't do anything
+ */
+class DefaultOnRetainInstanceListener : OnRetainInstanceListener {
+
+  override fun onChange(change: RetainInstanceChange) {}
```

Comment:
The default shouldn't be "empty", it should be showing notifications.

Reply:
This is  a `TODO` - move the notifications into default implementation and have TV implementations do only toasts. But it might take a while to do